doc/api/fs.md

```diff
@@ -334,7 +335,8 @@ added: v0.5.8
 
 * `error` {Error}
 
-Emitted when an error occurs while watching the file.
+Emitted when an error occurs while watching the file. The errored
```

Comment:
Maybe we should also documents that the errored watcher will be closed by core? I have not looked into the user land patterns yet but I am guessing they hit the crashes because they attempt to close the watcher in the error event handler, even though we have been closing the errorred watchers ourselves.